accepted/2020/mono-convergence/globalization-support-in-.net5.md

```diff
@@ -0,0 +1,82 @@
+# Globalization Support in .NET 5
+
+## Overview
+Globalization in the .NET Core space has been handled by taking a dependency on the [icu4c](http://site.icu-project.org/home) library and p-invoking into it from managed code.  This model has been successful for desktop and server modes mostly because they can afford to take on the disk space ICU requires.   However, in the size restricted workloads, this is quite a different story as there are differences in platform support and tolerance of taking on a significantly sized dependency.
+
+The purpose of this document is to describe what globalization support, if any, Android, iOS/tvOS, and WebAssembly have and what options there are should ICU not suffice.
+
+## .NET Core ICU dependencies
+All of the existing ICU dependencies in .NET Core today are located inside System.Private.CoreLib. The .NET Core does not install ICU and relies on the unmanaged implementation to be available on the system.
+
+The existing implementation requires ICU version is [50](https://github.com/dotnet/runtime/blob/4480bdbe66c55caceb64fc3477010435ba70c4ec/src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.c#L60) or higher.
+
+The implementation files which reference globalization interop are:
+
+* [System.Private.CoreLib/src/System/Globalization/IdnMapping.Unix.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Globalization/IdnMapping.Unix.cs)
+* [System.Private.CoreLib/src/System/Globalization/CalendarData.Unix.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.Unix.cs)
+* [System.Private.CoreLib/src/System/Globalization/CultureData.Unix.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Unix.cs)
+* [System.Private.CoreLib/src/System/Globalization/TextInfo.Unix.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Globalization/TextInfo.Unix.cs)
+* [System.Private.CoreLib/src/System/Globalization/CompareInfo.Unix.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.Unix.cs)
+* [System.Private.CoreLib/src/System/Globalization/JapaneseCalendar.Unix.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Globalization/JapaneseCalendar.Unix.cs)
```

Comment:
>*  [](start = 0, length = 2)

I believe TimeZoneInfo.Unic.cs is also has dependency on ICU 
[https://github.com/dotnet/runtime/blob/88cc71b26ba1eb430aaeeebddcd5675b3009f2ee/src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs#L127](https://github.com/dotnet/runtime/blob/88cc71b26ba1eb430aaeeebddcd5675b3009f2ee/src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs#L127)